rome.json

```diff
@@ -19,8 +19,7 @@
       ".vscode-test",
       "dist/**",
       "out/**",
-      "src/providerRuntime/index.ts",
       "*.json"
     ]
   }
-}
+}
```

Comment:
What you did to my newline? :p

Reply:
it was just on vacation it's coming back